action/__tests__/stoatApiHelpers.test.ts

```diff
@@ -0,0 +1,59 @@
+import * as core from '@actions/core';
+import { beforeEach, describe, expect, it, jest } from '@jest/globals';
+import fetch, { Response } from 'cross-fetch';
+
+import { PROD_API_URL_BASE, getApiUrlBase } from '../src/stoatApiHelpers';
+
+jest.mock('cross-fetch', () => jest.fn());
+const mockFetch = fetch as jest.MockedFunction<typeof fetch>;
```

Comment:
TIL

Reply:
I figured out that as long as the code and the jest test imports `fetch` from the same module (`cross-fetch` here), the mock works. Previously it did not work because I was importing `fetch` from `node-fetch`.